Task2/operations.js

```diff
@@ -0,0 +1,46 @@
+
+function checkMoreArgsError(arguments) {
+    if (arguments.length > 5) {
+        throw new Error("Only 2 numbers should be passed as parameters");
+    }
+}
+function checkNumberError(number) {
+    if (isNaN(number))
+        throw new Error(`Given argument ${number} is not a number, Please provide valid input`);
+}
+
+module.exports.addition = arguments => {
+
+    return arguments.slice(3).reduce((sum, current) => {
```

Comment:
it need to be slice(2) or slice of (3) 

why we do slice here , explain in comments  and why specifically 3rd index as per your answer

Reply:
arguments[0] will be The file path of folder
argument[1] will be the path of file
arguments[2] will be Operation(addition, subtraction,  multiplication or division
from arguments[3] the numbers array will be started, So I'm slicing the array from index 3.